### PR TITLE
Modify Comparison page

### PR DESCRIPTION
**Pull request by ivan-bonk**
- create feature folder in store
    - created a separate folder for each component
    - remove default export
    - remove anonymous functions
    - add to each component own styles and interface

### CODE REVIEW

**Review by Pashkaua — APPROVED** [review]
Nice !

File: src/pages/comparison/components/characteristics/characteristics.component.tsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
-import { ComparisonElement } from './comparison-element.component';
-import '../comparison.scss';
-import { CharacteristicProps, PetToCompare } from '../interfaces/pet.interface';
+import { ComparisonElement } from '../comparison-element/comparison-element.component';
+import '../../comparison.scss';
```
Comment (enheit) [inline src/pages/comparison/components/characteristics/characteristics.component.tsx:3]: Doesn't look like CSS modules :) Take a look at this part

**Review by Danik1702 — APPROVED** [review]
Good job!

File: src/pages/comparison/components/comparison-element/comparison-element.component.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 import { Link } from 'react-router-dom';
-import { Profile } from './profile.component';
-import { Rate } from './rate.component';
-import { ROUTES } from '../../../shared/constants/routes.constants';
+import { Profile } from '../profile/profile.component';
+import { Rate } from '../rate/rate.component';
+import { ROUTES } from '../../../../shared/constants/routes.constants';
 
-import '../comparison.scss';
+import './comparison-element.scss';
```
Comment (enheit) [inline src/pages/comparison/components/comparison-element/comparison-element.component.tsx:7]: Doesn't look like CSS modules. Will you use it?

Comment (ivan-bonk) [reply src/pages/comparison/components/comparison-element/comparison-element.component.tsx:7]: Yes, i remember this.
I will add CSS modules in next pull request

File: src/pages/comparison/components/characteristics/characteristics.component.tsx (hunk 1 above)
Comment (ivan-bonk) [reply src/pages/comparison/components/characteristics/characteristics.component.tsx:3]: Agree with you

**Review by enheit — APPROVED** [review]

**Pull request merged**